features/foutafhandeling.feature

```diff
@@ -11,31 +11,30 @@ Functionaliteit: Afhandeling van fouten
   In de foutresponse krijgt "instance" de url van het request die tot de fout heeft geleid.
 
   We kennen de volgende foutsituaties:
-  | Foutsituatie                        | status | voorbeeld title                                                   | code              |
-  | Geen parameter is meegegeven        | 400    | Geef tenminste één parameter op.                                  | paramsRequired    |
-  | Verplichte parameter(combinatie)    | 400    | Minimale combinatie van parameters moet worden opgegeven.         | paramsCombination |
-  | Niet toegestane parametercombinatie | 400    | De combinatie van opgegeven parameters is niet toegestaan.        | unsupportedCombi  |
-  | Parametervalidatie                  | 400    | Een of meerdere parameters zijn niet correct.                     | paramsValidation  |
-  | Teveel zoekresultaten               | 400    | Teveel zoekresultaten.                                            | tooManyResults    |
-  | Niet geauthenticeerd                | 401    | Niet correct geauthenticeerd.                                     | authentication    |
-  | Geen autorisatie voor operatie      | 403    | U bent niet geautoriseerd voor deze operatie.                     | autorisation      |
-  | Opgevraagde resource bestaat niet   | 404    | Opgevraagde resource bestaat niet.                                | notFound          |
-  | Accept header <> JSON               | 406    | Gevraagde contenttype wordt niet ondersteund.                     | notAcceptable     |
-  | Accept-Crs header niet ondersteund  | 406    | Gevraagde coördinatenstelsel {crs} wordt niet ondersteund.        | crsNotAcceptable  |
-  | Geen Content-Crs opgegeven          | 412    | Coördinatenstelsel van gestuurde geometrie moet worden opgegeven. | contentCrsMissing |
-  | Geen Accept-Crs opgegeven           | 412    | Gewenste coördinatenstelsel voor geometrie moet worden opgegeven. | acceptCrsMissing  |
-  | Crs wordt niet ondersteund          | 415    | Coördinatenstelsel {crs} in Content-Crs wordt niet ondersteund.   | crsNotSupported   |
-  | Technische of interne fout          | 500    | Interne server fout.                                              | serverError       |
-  | Bronservice is niet beschikbaar     | 503    | Bronservice {bron} is niet beschikbaar.                           | sourceUnavailable |
-  | Raadplegen geeft meerdere personen  | 400    | Opgegeven {parameternaam} is niet uniek.                          | notUnique         |
+  | Foutsituatie                          | status | voorbeeld title                                            | code              |
+  | Geen verplichte parameters meegegeven | 400    | Geef de verplichte parameter(s) op                         | paramsRequired    |
```

Comment:
Deze fout is toegevoegd maar heeft dezelfde code als de volgende fout (die al bestond). Het lijkt mij dat de code uniek moet zijn.